src/libecalc/dto/components.py

```diff
@@ -364,15 +363,6 @@ def check_user_defined_category(cls, user_defined_category, info: ValidationInfo
 
         return user_defined_category
 
-    @field_validator("fuel_consumers", mode="before")
```

Comment:
Check that either fuel_consumer or venting_emitters is set? Would be nice to also add that in the yaml class

Reply:
@jsolaas , you mean in the yaml_installation right? So, add a check here and in yaml class (even if it is not in use yet)?